task/sql-tasks.js

```diff
@@ -177,7 +280,12 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT 
+            COUNT(ProductID) as TotalOfCurrentProducts, sum(Discontinued) as TotalOfDiscontinuedProducts

```

Comment:
**sum(Discontinued)** - use count when you need to count records, it better shows intent

 it's better to write this query like this
notice that from-clause is optional.



```
select
   (select count()...) as 'T'
   (select count()...) as 'TT'
```